boards/x86/qemu_x86/qemu_x86_tiny.ld

```diff
@@ -54,6 +54,12 @@
 #define APP_SHARED_ALIGN		MMU_PAGE_ALIGN_PERM
 #endif
 
+/* Located in generated directory. This file is populated by calling
+ * zephyr_linker_sources(DEFINES ...). This typically contains the vector
```

Comment:
All these comments are incorrect, the `DEFINES` snippet does not typically contain the vector table/debug info

Reply:
Thanks for pointing out, fixed.